src/solvers/refinement/string_constraint_instantiation.cpp

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************\
+
+Module: Defines related function for string constraints.
+
+Author: Jesse Sigal, jesse.sigal@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Defines related function for string constraints.
+
+#include <solvers/refinement/string_constraint_instantiation.h>
+
+#include <solvers/refinement/string_constraint.h>
+#include <solvers/refinement/string_constraint_generator.h>
+#include <solvers/refinement/string_refinement.h>
+
+/// Instantiates a quantified formula representing `not_contains` by
+/// substituting the quantifiers and generating axioms.
+/// \related string_refinementt
+/// \param [in] axiom: the axiom to instantiate
+/// \param [in] index_set0: the index set for `axiom.s0()`
+/// \param [in] index_set1: the index set for `axiom.s1()`
+/// \param [in] generator: generator to be used to get `axiom`'s witness
+/// \param [out] lemmas: the lemmas produced through instantiation
```

Comment:
If it's really the case, it should be the return value of the function. But if you want to append to an existing list, this should be documented accordingly.